portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +27,96 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+//*********************** LOADING SCREEN ***********************
+function setVisible(selector, bool) {
+    const el = document.getElementById(selector);
+    el.classList.toggle('hiddenclass', bool);
+}
+
+function swapLoaderForContent(){
+    setVisible('header', false);
+    setVisible('content', false);
+    setVisible('loaderid', true);
+}
+
+//*********************** SEARCH BAR FUNCTIONS ***********************
+
+window.onload = function(){
+    swapLoaderForContent();
+    document.getElementById('searchbar').onkeydown = function(e){
+        if(e.keyCode == 13){
+            runSearch(document.getElementById('searchbar').value.toLocaleLowerCase());
```

Comment:
nit: break apart larger lines so they fit under 80 characters:
```
    runSearch(
      document.getElementById('searchbar').value.toLocaleLowerCase());
```

You can also break apart at `.` characters:
```
    runSearch(
      document.getElementById('searchbar')
        .value.toLocaleLowerCase());
```